packages/ui/src/memberships/modals/BuyMembershipModal/BuyMembershipSignModal.tsx

```diff
@@ -97,8 +97,10 @@ export const BuyMembershipSignModal = ({
           <Row>
             <Info>
               <TextMedium>
-                Tokens subject to Vesting and Invitation locks do not cover membership creation fees, and cannot be used
-                to purchase new memberships.
+                Invitation lock can be spent on transaction fees and staking for proposals, voting and working groups
```

Comment:
In light of #3321 i'm wondering if it should be more clear like `can ONLY be spent on`
> show some very clear big visual that sort of says "you will get some tiny amount of free coins, but you can only use them for fees, not transfer or selling, that wont work"